azure_monitor/django_sample/readme.md

```diff
@@ -1,65 +1,30 @@
-# Capture Dependencies
+---
+title: 'Sample: Configuring Django applications with OpenCensus'
+description: Configuring Django applications with OpenCensus.
+author: givenscj
+ms.author: givenscj
+ms.devlang: python
+ms.topic: sample
+ms.date: 07/27/2022
+ms.custom: []
+---
+
+# Sample: Configuring Django applications with OpenCensus
 
```

Comment:
We should probably remove all the files in the `django_sample` folder that are not needed for this sample to run right? Like `.azure` and `.github` folders.